CraftStore_Events.lua

```diff
@@ -175,6 +175,20 @@ function CS.OnReticleHiddenUpdate(eventCode,hidden)
 end
 
 function CS.OnPlayerActivated(eventCode,initial)
+  if CS.Debug then zo_callLater(function() CHAT_ROUTER:AddSystemMessage("CS.Account.crafting.jewelryIdSwapMigrationAlreadyDone: "..tostring(CS.Account.crafting.jewelryIdSwapMigrationAlreadyDone)) end, 50) end
+  if CS.Account.crafting.jewelryIdSwapMigrationAlreadyDoneDone == nil or CS.Account.crafting.jewelryIdSwapMigrationAlreadyDoneDone == false then
+    zo_callLater(
+            function()
+                CHAT_ROUTER:AddSystemMessage("[CraftStore] The internal indices of the game for jewelry (rings and necklaces) got swapped by ZOS. CraftStore need to migrate its SavedVariables once. Starting...")
+            end, 50)
+    CS.MigrateJewelryIdSwap()
+    CS.Account.crafting.jewelryIdSwapMigrationAlreadyDone = true
```

Comment:
Shouldn't this be DoneDone as well?

Reply:
Oops. You're totally right.. My head was in the sky it seems.

Reply:
Changed